src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -72,6 +72,16 @@ public function describe(OutputInterface $output, $object, array $options = arra
     }
 
     /**
+     * Returns the output.
+     *
+     * @return OutputInterface The output
+     */
+    protected function getOutput()
```

Comment:
Changing the visibility makes it mutable. The methods allows us to keep the property read-only.